templates/server-ingress.yaml

```diff
@@ -2,6 +2,9 @@
 {{- if ne .mode "external" }}
 {{- if .Values.server.ingress.enabled -}}
 {{- $serviceName := include "vault.fullname" . -}}
+{{- if and (eq .mode "ha" ) (and (eq (.Values.server.service.enabled | toString) "true" ) (eq (.Values.global.enabled | toString) "true")) }}
```

Comment:
This change would only allow ingress to be created for HA mode, but wouldn't support dev or standalone.  This logic will need to be expanded to support the other modes as well.

Reply:
no it wouldn't. this is specifically appending "-active" to the serviceName defined on line 4, if and only if it's ha. That doesn't mean there won't be an ingress in the other cases, just that it won't include "-active" on the service name

Reply:
Aha, I see now.  Thanks!